config.yml

```diff
@@ -89,12 +102,11 @@ WISTIA:
 ROCK_MAPPINGS:
   INTERACTIONS:
     # Name of the InteractionChannel that will be created
-    CHANNEL_NAME: Apollos NewSpring App
+    CHANNEL_NAME: Apollos App
     # Name of the InteractionComponent that will be created for Content Items or Prayer Requests
-    COMPONENT_NAME: Apollos NewSpring Content Item
```

Comment:
Not sure these should have changed?

Reply:
I just removed "Newspring" from the interaction channel, seems redundant

Reply:
I guess you could convince me either way, but do you think this should say "NewSpring" and not "Apollos" if this is going to be in our Rock and our Interactions?

Reply:
seems silly to label it "newspring" on Newspring's rock. honestly it could just be "App" but core says "Apollos App" so I just matched it.

Reply:
I don't hate that. Another question this makes me think of though: will changing the name mess with the current set of interactions we have in Rock? Meaning ... will we now have more than one channel/component where interaction data is being written? One prior to this change and one after?

Reply:
Changing this will mean two different channels/components. I would recommend sticking with your old naming, if you want to avoid that. 

We changed this on core so the channel names would be more clear going forward.